simplified-books-controller/src/main/java/org/nypl/simplified/books/controller/ProfileFeedTask.kt

```diff
@@ -282,20 +282,29 @@ internal class ProfileFeedTask(
         values.filter { book -> book.book.account == accountID }
       } else {
         values
-      }.filter {
-        this.accountIsLoggedIn(it.book.account)
+      }.filter { bookStatus ->
+        if (!this.requiresLogin(bookStatus.book.account)) {
+          bookStatus.status is BookStatus.Loaned.LoanedDownloaded
```

Comment:
I'm not sure I understand why `LoanedDownloaded` specifically. Why not `Loaned`?

Reply:
If it was just `Loaned`, it would have the same effect as it was before. I filtered the `LoanedDownloaded` books for the free accounts (the ones that don't require login) because if you deleted a book it would stay on your "My Books" tab until you restarted the app. In this case, it will just show the downloaded books for this type of accounts and won't show the other ones.

I may be looking at this the wrong way, but I performed some tests and it seemed fine to me!